### ISSUE TITLE
Image, Embed problem

### ISSUE THREAD

**Issue by ZeroCho** [jpuri/react-draft-wysiwyg#75]
@jpuri Thanks for the great editor!
When I embed url or add image url, img and iframe src is not the url I typed but **[object Event]**
`<img src="[object Event]" role="presentation">`
`<iframe width="100%" height="315px" src="[object Event]" frameborder="0" allowfullscreen=""></iframe>`
I found out that it is due to [here](https://github.com/jpuri/react-draft-wysiwyg/blob/master/js/src/components/EmbeddedControl/index.js#L23) and [here](https://github.com/jpuri/react-draft-wysiwyg/blob/master/js/src/components/ImageControl/index.js#L50)
I think the order should be changed into this
`const src = this.state.imgSrc || imgSrc;`
also
`const link = this.state.embeddedLink || embeddedLink;`
I'm not sure what do those **imgSrc** and **embeddedLink** do. As an eventHandler callback, do we need second parameter?

**Comment by jpuri** [comment]
@ZeroCho: the image upload callback is required to return result in specific format. Please check docs for it. Let me know if you need more help.

**Comment by ZeroCho** [comment]
@jpuri then, why I got **[object Event]** instead of url?

**Comment by jpuri** [comment]
Apparently your callback is not returning result  in required format.

**Comment by ZeroCho** [comment]
Hmm.. That's weird. I don't use any image upload callback nor custom options. I will console log what happens and tell you.

**Comment by ZeroCho** [comment]
This is how I did.
1. I cloned your repository and run **npm start** which use your playground.
2. I put some image url and console.logged event, imgSrc and this.state.imgSrc
3. event == Proxy,  imgSrc == Event, this.state.imgSrc == url

**Comment by jpuri** [comment]
Strange I do not face issue with images. Which browser are you using ?

**Comment by ZeroCho** [comment]
Its chrome54 windows10 nodejs7.1 npm4.0.3
Its is the first time that i faced this too... very embarassed.

**Comment by EpsApps** [comment]
i had this issue to. 

To fix it, go to node_modules/react-draft-wysiwyg/dist/react-draft-wysiwyg.js and find:

```Javascript
r.addImage=function(t,e){var n=r.props,o=n.editorState,i=n.onChange,a=e||r.state.imgSrc,u=l.Entity.create("IMAGE","MUTABLE",{src:a}),s=l.AtomicBlockUtils.insertAtomicBlock(o,u," ");i(s),r.toggleModal()},
```

replace the code above with:

```Javascript
r.addImage=function(t,e){
    var n=r.props;
    var o=n.editorState;
    var i=n.onChange;
    var a = (typeof e === 'string') ? e : r.state.imgSrc;
    var u=l.Entity.create("IMAGE","MUTABLE",{src:a});
    var s=l.AtomicBlockUtils.insertAtomicBlock(o,u," ");
    i(s),r.toggleModal()
},
```

**Comment by ZeroCho** [comment]
@jpuri Any updated news for this? I think just changing the order of variables as I suggested can solve this problem without affecting others.

**Comment by jpuri** [comment]
@ZeroCho, I am working on it now. I will also look at your PR, thanks for your patience :)

**Comment by jpuri** [comment]
@ZeroCho: can you also plz share with me what exactly is returned from callback you have written ad what it gives after its resolved.

**Comment by jpuri** [comment]
@ZeroCho: I could now see what issue you are mentioning. Thanks a lot for pointing out - I am wondering how I missed to find it. You will have the fix release by tomorrow.

**Comment by jpuri** [comment]
Issue has been fixed and release in version `1.2.0`.
